ghostly.py

```diff
@@ -97,6 +97,16 @@ def load(self, url):
         self.browser.get(url)
 
 
+    def navigate(self, navigation):
+        """
+        Navigate with browser history
+        http://selenium-python.readthedocs.org/navigating.html#navigation-history-and-location
+        TODO: handle all the methods available!
+        back, forward, refresh, to(navigation)
+        """
+        self.browser.back()
```

Comment:
@skumo okay, let's make separate back / forward / refresh commands. Keeping it SIMPLE.